content/docs/5.1/concepts/explore-tikv-features/distributed-transaction.md

```diff
@@ -11,21 +11,22 @@ This chapter walks you through a simple demonstration of how TiKV's distributed
 
 ## Prerequisites
 
-Please start a TiKV Cluster and install tikv-client python package according to [TiKV in 5 Minutes](../../tikv-in-5-minutes).
+Before you start, ensure that you set up a TiKV Cluster and install tikv-client python package according to [TiKV in 5 Minutes](../../tikv-in-5-minutes).
 
 {{< warning >}}
-TiKV Java Client's Transaction API is not released, so Python Client example is used here.
+TiKV Java Client's Transaction API has not been released yet, so Python Client example is used here.
 {{< /warning >}}
 
-## Step 1: Test snapshot isolation
+## Test snapshot isolation
 
 Transaction isolation is one of the foundations of database transaction processing. Isolation is one of the four key properties of a transaction (commonly referred as ACID).
 
-TiKV implements [Snapshot Isolation (SI)](https://en.wikipedia.org/wiki/Snapshot_isolation) consistency, which means
-- all reads made in a transaction will see a consistent snapshot of the database (in practice it reads the last committed values that existed at the time it started)
-- the transaction itself will successfully commit only if no updates it has made conflict with any concurrent updates made since that snapshot
+TiKV implements [Snapshot Isolation (SI)](https://en.wikipedia.org/wiki/Snapshot_isolation) consistency, which means:
 
-Let's try the following example to test TiKV's snapshot isolation.
+- all reads made in a transaction will see a consistent snapshot of the database (in practice it reads the last committed values that existed at the time it started);
+- the transaction itself will successfully commit only if no updates it has made conflict with any concurrent updates made since that snapshot.
```

Comment:
@marsishandsome "no updates it has made conflict with any concurrent updates made since that snapshot" looks confusing. Could you try to describe it in Chinese?